### DISCUSSION TITLE
Allow css styling per page (automatic <body> class)

### DISCUSSION THREAD

**Discussion by underdpt** [filamentphp/filament#8644] [Ideas]
Hello,

I've been trying to add a background image to the login page through a renderhook, and found that it's shown on all pages:

```php
->renderHook(
    'panels::body.start',
    fn () => Blade::render('
        <div
            class="background w-full h-full fixed"
            style="z-index: -999; background-image: url(\'/img/fondo.jpg\')"
        ></div>'
    )
```

As currently renderhooks cannot be scoped to non-resources pages (discussion #8212) I wonder if it would be possible to add <body> classes to allow further theme customization. For example, when on login page, it could be:

```html
<body class="fi-auth-login ...">
```
And on some resource (User edit):

```html
<body class="fi-resource-edit-user ...">
```

This would allow to easily  add css that only affects some pages.

**Comment by krekas** [comment]
Login page uses the simple page and it has css classes which you can use. Just checked them using developer tool
![image](https://github.com/filamentphp/filament/assets/11015977/3c2150d2-0f10-43ca-95df-57bd35f9b18c)

**Comment by underdpt** [comment]
Yes, I know, but with those css classes it's impossible to style the `<body>` tag and also to distinguish between pages using that layout, so all would be styled the same, which might not be what the developer wants.
